test/controllers/changesets_controller_test.rb

```diff
@@ -131,7 +145,7 @@ def test_index_user_not_found
   def test_index_friends
     private_user = create(:user, :data_public => true)
     friendship = create(:friendship, :befriender => private_user)
-    create(:changeset, :user => friendship.befriendee)
+    changeset = create(:changeset, :user => friendship.befriendee, :num_changes => 1)
```

Comment:
Shouldn't this create some "other" changesets to make sure we only get the expected one?

Reply:
Done! Thanks for picking this up